core/src/main/scala/cats/data/Validated.scala

```diff
@@ -357,28 +388,6 @@ private[data] sealed abstract class ValidatedInstances extends ValidatedInstance
 
       override def isEmpty[A](fa: Validated[E, A]): Boolean = fa.isInvalid
     }
-    // scalastyle:on method.length
```

Comment:
We may want to keep this `// scalastyle:off method.length` comment.

At the same time I think we can remove these comments around the `ApplicativeError` instance (`catsDataApplicativeErrorForValidated`).